client/wc-api/onboarding/operations.js

```diff
@@ -156,29 +166,36 @@ function getPluginErrorMessage( action, plugin ) {
 				__( 'There was an error installing %s. Please try again.', 'woocommerce-admin' ),
 				pluginName
 			)
-		: sprintf(
-				__( 'There was an error activating %s. Please try again.', 'woocommerce-admin' ),
-				pluginName
-			);
+		: __( 'There was an error activating your plugins. Please try again.', 'woocommerce-admin' );
 }
 
 function installPlugins( resourceNames, data, fetch ) {
 	const resourceName = 'plugin-install';
-
 	if ( resourceNames.includes( resourceName ) ) {
 		const plugins = data[ resourceName ];
-		return doPluginActions( fetch, 'install', plugins );
-	}
-
-	return [];
-}
-
-function activatePlugins( resourceNames, data, fetch ) {
-	const resourceName = 'plugin-activate';
 
-	if ( resourceNames.includes( resourceName ) ) {
-		const plugins = data[ resourceName ];
-		return doPluginActions( fetch, 'activate', plugins );
+		return plugins.map( async plugin => {
+			return fetch( {
+				path: `${ NAMESPACE }/onboarding/plugins/install`,
+				method: 'POST',
+				data: {
+					plugin,
```

Comment:
The WP dashboard allows activating multiple plugins at once but will only install one at a time. Although this could be revisited, it would probably be best to use the core limits.

Reply:
I thought about this as I was updating the activate endpoint. I was worried about the potential of an endpoint timing out and doing too much. The install has to pull the download, unzip, and install. I think it might be better to keep these separate for now.

Reply:
That seems reasonable.